pi.py

```diff
@@ -1 +1,578 @@
-test=1
+# Potential Intensity Calculation for Python
+# -----------------------------------------------------------------------------------
+# Adapted from pcmin.m by Kerry Emanuel (ftp://texmex.mit.edu/pub/emanuel/TCMAX)
+# Originally Updated by Daniel Gilford for
+# Gilford et al. (2017) -- https://journals.ametsoc.org/doi/full/10.1175/JCLI-D-16-0827.1
+# Gilford et al. (2019) -- https://journals.ametsoc.org/doi/10.1175/MWR-D-19-0021.1
+# 
+# Adapted for Python (pyPI) by Daniel Gilford, PhD (Rutgers U., daniel.gilford@rutgers.edu)
+# Last updated 4/14/2020
+# -----------------------------------------------------------------------------------
+#
+# Revision History:
+#   Revised on 9/24/2005 by K. Emanuel to fix convergence problems at high pressure
+#     --Converted to MATLAB  5/17/2008
+#   Revised 7/20/15 by D. Gilford to output the LNB
+#   Revised 8/4/16 by D. Gilford to include lack of convergence if SST < 5C for TO/LNB
+#   Revised 8/5/16 by D. Gilford to fix the "cape()" function output and include LNB
+#   Revised 10/3/16 by D. Gilford to set LNB to the pressure-weighted crossing of buoyancy from negative to positive (the zero-line)
+#   Revised 1/7/2017 by Luke Davis to fix and improper definition of JMIN, which should be the lowest profile level at or above the parcel level in the calculation of CAPE. Kerry is grateful to Luke and to Tim Merlis for pointing out this error. In practice, the error was usually less than 1 hPa in central pressure.    
```

Comment:
Nevermind I see it here (see comment in pc_min.m).

Reply:
:)